packages/delayedInbox-l2msg/README.md

```diff
@@ -0,0 +1,38 @@
+# delayedInbox-l2msg Tutorial
+
+(Noted this can only be done in nitro stack, not in arbitrum classic, so l2 rpc please use nitro test rpc)
```

Comment:
(Note this can only be done in nitro stack, not in Arbitrum classic. So please use the Nitro test rpc for the L2 rpc.)

Reply:
We already set the L2 rpc to the nitro test rpc in the env sample. So maybe we can say sth like:
(Note this can only be done in nitro stack, not in Arbitrum classic. That's the reason the L2 rpc is set to the nitro test rpc in the env sample.)
I guess it makes it less confusing for users.

Reply:
Since our testnets are all nitro stack, so i just made it to (Note this can only be done in nitro stack, not in Arbitrum classic.)